src/commands.ts

```diff
@@ -418,35 +440,40 @@ export class Commands {
 		console.log("Asset updated " + txid);
 	}
 
-	public async editAsset(args: string[]) {
-		const asset = await this.aquarius.waitForAqua(args[1]);
-		if (!asset) {
-			console.error(
-				"Error fetching DDO " + args[1] + ".  Does this asset exists?"
-			);
-			return;
-		}
-		let updateJson;
-		try {
-			updateJson = JSON.parse(fs.readFileSync(args[1], "utf8"));
-		} catch (e) {
-			console.error("Cannot read metadata from " + args[1]);
-			console.error(e);
-			return;
-		}
-		// Get keys and values
-		const keys = Object.keys(updateJson);
+	public async getJobStatus(args: string[]) {
+		const jobStatus = (await ProviderInstance.computeStatus(
+			this.providerUrl,
+			await this.signer.getAddress(),
+			args[2],
+			args[1]
+		)) as ComputeJob;
+		console.log(jobStatus);
+	}
 
-		for (const key in keys) {
-			asset[key] = updateJson[key];
-		}
+	public async getJobResults(args: string[]) {
+		const jobStatus: ComputeJob[] = (await ProviderInstance.computeStatus(
+			this.providerUrl,
+			await this.signer.getAddress(),
+			args[2],
+			args[1]
+		)) as ComputeJob[];
+		console.log(jobStatus);
```

Comment:
this is redundant, but we can remove it later

Reply:
yep aggred, removed the method and updated job status logs to display inner objects as well